### ISSUE TITLE
Regression in indentation of multiline conditions

### ISSUE THREAD

**Issue by f18m** [bemeurer/beautysh#42]
I noticed that with latest master the following code block 

```
    # do basic argument checks
    if [ "$A" == "true" ] || [ "$A" == "false" ] || [ "$A" == "noconflicts" ] || \
        [ "$B" == "true" ] || [ "$B" == "false" ] || [ "$B" == "noconflicts" ] || \
        [ "$C" == "true" ] || [ "$C" == "false" ]; then
        printerr "Internal error: invalid function call argument"
        exit 2
    fi
```

is now indented as:

```
    # do basic argument checks
    if [ "$A" == "true" ] || [ "$A" == "false" ] || [ "$A" == "noconflicts" ] || \
        [ "$B" == "true" ] || [ "$B" == "false" ] || [ "$B" == "noconflicts" ] || \
    [ "$C" == "true" ] || [ "$C" == "false" ]; then
        printerr "Internal error: invalid function call argument"
        exit 2
    fi
```

3rd line has wrong indentation. I think this is perhaps a regression introduced with one of my latest patches, so I'm opening this to avoid forgetting.

**Comment by bemeurer** [comment]
I'm currently working on a redesign of the beatifying process, it's taking place in the parser-redesign branch. I'm hoping to use [bashlex](https://github.com/idank/bashlex) to actually process the files, which should make things much simpler and avoid errors like these!

**Comment by f18m** [comment]
Hi @bemeurer ,
good to know! do you think it makes sense then to work on a fix on the current "indentation engine" or not?
Also are you planning a new release before this bashlex reworking?
It would be very useful for me to have the --force-function-style and --version in a released build, thanks :)

**Comment by bemeurer** [comment]
I think fixing this version would absolutely be useful @f18m, the rework is mostly experimental, and may or may not work.

Yeah, I indent on releasing 4.0 with those changes as soon as this gets fixed :).

**Comment by bemeurer** [comment]
So, can you confirm that this was fixed by #43?

**Comment by f18m** [comment]
Yes I conform it's fixed. Now the block of code where the regression happened Is also in the unit tests :)
